.github/workflows/deploy.yml

```diff
@@ -113,6 +122,19 @@ jobs:
         name: distributions
         path: dist
 
+    - name: List distributions to be deployed
+      run: ls -l dist/
+
+    - name: Verify sdist artifact attestation
+      env:
+        GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: gh attestation verify dist/awkward-*.tar.gz --repo ${{ github.repository }}
+
+    - name: Verify wheel artifact attestation
+      env:
+        GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: gh attestation verify dist/awkward-*.whl --repo ${{ github.repository }}
```

Comment:
A note related to https://github.com/cli/cli/issues/9215, this is only working currently because there is one wheel. `gh attestation verify` can't operate on file globs!